src/Tribe/Cost_Utils.php

```diff
@@ -35,11 +35,11 @@ public function get_all_costs() {
 		$costs = $wpdb->get_col( "
 			SELECT DISTINCT meta_value
 			FROM   {$wpdb->postmeta}
-			WHERE  meta_key = '_EventCost'
-			       AND LENGTH( meta_value ) > 0
-		" );
+			WHERE
+				meta_key = '_EventCost' AND
+				LENGTH( meta_value ) > 0;" );
```

Comment:
Unless you aren't seeing what I'm seeing:

![screen shot 2015-11-23 at 5 23 53 pm](https://cloud.githubusercontent.com/assets/430385/11351866/0511fd3e-9207-11e5-847a-b2b4bda2040a.png)